aries-site/src/examples/templates/list-views/ListOrderExample.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+
+import { Box, Button, List, Menu, Text } from 'grommet';
+
+const movies = [
+  { title: 'Ponyo', genre: 'Fantasy/Adventure' },
+  { title: 'Spritied Away', genre: 'Family/Fantasy' },
+  { title: 'Grave of the Fireflies', genre: 'Fantasy/Adventure' },
+  { title: "Wayne's World", genre: 'Comedy/Music' },
+  { title: 'Detroit Rock City', genre: 'Comedy/Music' },
+  { title: 'Home Alone 2', genre: 'Comedy/Family' },
+  { title: 'The Lion King', genre: 'Family/Musical' },
+  { title: 'Mulan', genre: 'Action/Adventure' },
+  { title: 'Inside Out', genre: 'Comedy/Family' },
+];
+
+export const ListOrderExample = () => {
+  const [ordered, setOrder] = useState(movies);
+  const [allowReorder, setAllowReorder] = useState(false);
+
+  return (
+    <Box align="center" pad="large" role="application">
```

Comment:
Can we have a background-front here so it's white background like the other examples?

Reply:
I'm also wondering if there's any designs from products that have this inline list re-ordering. That way, we could make this example a bit more aligned with a "real world" example

Reply:
Done